packages/react-components/src/components/utils/videoTileStylesUtils.tsx

```diff
@@ -10,3 +10,29 @@ export const getVideoTileOverrideColor = (isVideoRendered: boolean, theme: Theme
   // when video is being rendered, the info has a grey-ish background, so no use of theme
   return { color: isVideoRendered ? palette[color] : theme.palette[color] };
 };
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * @private
+ */
+export const getCurrentRelativeUnixTime = () => {
+  let baseTimeStamp = new Date();
+  baseTimeStamp.setMonth(0);
+  baseTimeStamp.setDate(1);
+  baseTimeStamp.setHours(0, 0, 0, 0);
+
+  let baseUnixTimestamp = Math.floor(baseTimeStamp.getTime() / 1000);
+  return Math.floor(new Date().getTime() / 1000) - baseUnixTimestamp;
+};
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * @private
```

Comment:
nit: add comment about this being temporary, sometimes temp code lives much longer than we want or focus shifts elsewhere and its not removed